monorepo/monorepo-ws/apps/federation-gateway/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:16.14-alpine
```

Comment:
jako možná by stálo za úvahu ty dockerfily sjednotit, ať nejsou čistě copy-paste. daj se kombinovat přes multistage například. nebo mít nějakej generickej a měnit argumentem buildu jen pár příkazů. 

Ale to je spíš na další task než to řešit v tomhle obřím MR.